apps-rendering/src/components/Standfirst/ImmersiveStandfirst.tsx

```diff
@@ -0,0 +1,24 @@
+import { css } from '@emotion/react';
+import { headline, remSpace } from '@guardian/source-foundations';
+import type { Item } from 'item';
+import { getFormat } from 'item';
+import DefaultStandfirst, { defaultStyles } from './Standfirst.defaults';
+
+interface Props {
+	item: Item;
+}
+
+const ImmersiveStandfirst: React.FC<Props> = ({ item }) => (
+	<DefaultStandfirst
+		item={item}
+		css={css(
+			defaultStyles(getFormat(item)),
+			css`
+				${headline.xsmall({ fontWeight: 'light' })}
+				margin-top: ${remSpace[3]};
+			`,
```

Comment:
I notice you've inlined styles like this (as opposed to storing them in a variable above) a few times. Are you finding this offers some advantages that may make us want to adopt it as a convention? Are you finding it easier to read the component at a glance like this for example?

Reply:
At the time I was thinking that it added the advantage of not having to check where the styles are being added in the markup, given that they're only a few lines in most cases. However in hindsight, given that the files are so much smaller now this advantage is pretty minimal and the loss of consistency outweighs it so I've reverted to a more conventional pattern